File: AKLanguageManager/iOS/DemoObjC/DemoObjC/SecondViewController.m
//
//  SecondViewController.m
//  DemoObjC
//
//  Created by Amr Koritem on 13/10/2022.
//

#import "SecondViewController.h"
@import AKLanguageManager;

@interface SecondViewController ()

@property (weak, nonatomic) IBOutlet UISegmentedControl *segmentedControl;

@end

@implementation SecondViewController

- (IBAction)changeSegmentImageDirection {
    [_segmentedControl revertImageHorizontalDirectionAt:1];
}

- (IBAction)changeButtonImageDirection:(UIButton *)sender {
    [sender revertImagesHorizontalDirection];
}

@end

File: AKLanguageManager/tvOS/DemoUIKit/DemoUIKit/SecondViewController.swift
//
//  SecondViewController.swift
//  DemoUIKit
//
//  Created by Amr Koritem on 19/10/2022.
//

import UIKit

class SecondViewController: UIViewController {
    @IBAction func changeButtonImageDirection(_ sender: UIButton) {
        sender.revertImagesHorizontalDirection()
    }
}
